src/react-aria-modal.js

```diff
@@ -268,39 +268,31 @@ function displace(WrappedComponent, options) {
       mounted: true
     };
 
-    constructor(props) {
-      super(props);
-
-      this.state = {
-        container: null,
-      };
-    }
+    container = null;
 
     componentDidMount() {
-      let container;
-
       if (options.renderTo) {
-        container = typeof options.renderTo === 'string' ? document.querySelector(options.renderTo) : options.renderTo;
+        this.container = typeof options.renderTo === 'string' ? document.querySelector(options.renderTo) : options.renderTo;
       } else {
-        container = document.createElement('div');
-        document.body.appendChild(container);
+        this.container = document.createElement('div');
+        document.body.appendChild(this.container);
       }
 
-      this.setState({ container });
+      this.forceUpdate();
     }
 
     componentWillUnmount() {
       if (!options.renderTo) {
-        this.state.container.parentNode.removeChild(this.state.container);
+        this.container.parentNode.removeChild(this.container);
```

Comment:
In rare cases, it's possible that `componentWillUnmount` will be called before `this.setState({ container })` applied changes to state. So it crashes when trying access `parentNode` property of `containter` state which is null.

I have replaced state by class property and added invocation of `this.forceUpdate()` in order to rerender component after container was mounted to DOM.